includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -628,6 +668,156 @@ private function get_section_carrier_dpd_pickup_options(): array
         ];
     }
 
+    /**
+     * Export defaults specifically for PostNL.
+     *
+     * @return array
+     */
+    private function get_section_carrier_postnl_export_defaults(): array
+    {
+        return [
+            [
+                "name"  => WCMP_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED,
+                "label" => __("Insured shipment (to €500)", "woocommerce-myparcelbe"),
```

Comment:
PostNL verzekering kan tot 2000 (van BE naar NL). De plugin even nalopen of alles dan wel klopt.